test/fs/test_fs_symlink_resolution.c

```diff
@@ -1,7 +1,5 @@
 #include <unistd.h>
 #include <fcntl.h>
-#include <fcntl.h>
-#include <unistd.h>
```

Comment:
This is an interesting one.     Do you know why it picked up on this but didn't remove unused includes in the rest of the codebase.   This somehow seems different fixing typos.

Reply:
Moved to #26024